_data/utilities.yml

```diff
@@ -22,6 +22,12 @@ websites:
       img: att.png
       tfa: No
 
+    - name: Blau.de
+      url: https://www.blau.de/
+      img: blau.png
+      facebook: blaumobilfunk
+      tfa: No
+
```

Comment:
Please add proper `lang` tag so that message requesting 2fa would be in the correct language.

Reply:
Updated 👍